src/deprecations.jl

```diff
@@ -1,8 +1,7 @@
 Base.@deprecate_binding DirichletBoundaryConditions ConstraintHandler
 Base.@deprecate_binding DirichletBoundaryCondition Dirichlet
 
-import Base: push!
-@deprecate push!(dh::AbstractDofHandler, args...) add!(dh, args...)
+@deprecate Base.push!(dh::AbstractDofHandler, args...) add!(dh, args...) false
```

Comment:
I have a vague recollection that this doesn't work which is why there was an explicit import before.

Reply:
Yes, seems like the required export_old is not available on 1.6
I guess we could change to
``` 
function Base.push!(dh::AbstractDofHandler, args...) 
    @warn "push!(dh::DofHandler, args...) is deprecated, use add!(dh, args...) instead" maxlog=1
    return add!(dh, args...)
end
```
what do you think?

Reply:
What was the problem with the existing code? I think for 1.0 we will just make all of these errors and then we can use the `Base.push!` pattern.

Reply:
> What was the problem with the existing code?

Just that `push!` could be overloaded anywhere in the codebase without explicitly calling `Base.push!` (such a well-known function, so shouldn't be a problem, but AFAIU that was the reason for preferring e.g. `using Base: push!` over `import Base: push!`?

But if we remove it, then it doesn't matter and I'll close this PR :)